### DISCUSSION TITLE
Site Architecture and Navigation

### DISCUSSION THREAD

**Discussion by hepplerj** [chnm/lasfera#38] [Ideas]
This document exists to discuss and clarify the structure of the La Sfera project.

## Current Structure

Currently the website is structured to mimic the existing website. That includes the following navigation elements: 

- Introduction
    - La Sfera
    - Explore
    - La Sfera Challenge
- About
    - Context
    - Author
- Background
    - Team News
    - Goals
    - Team
- Resources
    - Maps
    - Samples
    - Bibliography
    - Links

## Proposed Architecture

The consensus seems to be that we don't want to mimic the existing structure, but instead architecture the site to focus primarily on the data and data-driven components of the manuscript project. With this in mind, I'd suggest the following: 

- Introduction
- About
    - Project Team
    - Technical Specifications
- Manuscript
    - Poetic Text: This is the transcription of the poetic text, along with line codes, (future) ability to switch manuscript transcriptions/translations, and IIIF images.
    - Identified Manuscripts: This is the table of manuscript for each siglum that is available in the database. This table takes you to the detail page of a manuscript. The detail page of a single manuscript will provide metadata about the manuscript, links to IIIF manifests or photographs (and display these for viewing), and display related toponyms. The related toponyms are linked to single toponym views that allow you to inspect a single toponym and its related manuscripts. 
    - Gazetteer: This is an index of the toponyms that includes a filterable list of the available toponyms as well as a map interface to show the location of each toponym. Both of these interfaces allow a user to click on a single toponym to get details about it. The toponym detail view provides a list of its aliases (if any), a map of its location (if applicable), and a list of the manuscript that contain the toponym.

**Comment by lmorreale3** [comment]
Hi Jason-
Thanks so much for this and sorry for the delayed reaction! I think in the summer we are all here and there. 
I think what you have proposed is good for the moment, but I would like to think about it a little more, and to talk it through with the others. For example, while the manuscripts indeed provide the data for what we are presenting, I don't really want that to be front-and-center as a guiding category. It's as if you were talking about "The Great Gatsby" and your main heading was "various printings." Anyway, we want to come up with categories that deal with content first, then with transmission second, I think, and this will take some brainstorming. So if we can do that on Wednesday, I think we can come back with a clearer sense of the architecture then.

**Comment by hepplerj** [comment]
Revision of architecture: 

Edition (renamed from Manuscript):
- Text, which includes:
   - Italian text w/ line codes -> corresponding image
   - English text w/ line codes -> corresponding image
   - Image
   - Map (modern cartographic rep. of toponyms)

Gazetteer: search fields, interactive map

Manuscript Tradition / Story of La Sfera / Forgotten Oceans / Manuscripts and their Makers (renamed from Identified Manuscripts): table of manuscripts, etc. -- wordsmithing tbd

Home | Edition | Gazetteer | Manuscript Tradition | About
